pom.xml

```diff
@@ -43,6 +43,35 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency> <!-- used to connect to sheets API -->
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client</artifactId>
+      <version>1.30.9</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client-appengine</artifactId>
+      <version>1.30.9</version>
+     </dependency>
+
```

Comment:
Can you add a comment to indicate what this dependency is for?

Reply:
Sure, it helps with the sheets integration using appengine.